tcms/testruns/static/testruns/js/get.js

```diff
@@ -291,12 +291,26 @@ function fileBugFromExecution(execution) {
         const trackerId = $('.one-click-bug-report-form #id-issue-tracker').val()
         jsonRPC('Bug.report', [execution.id, trackerId], result => {
 
+            // close the modal
+            $('#one-click-bug-report-modal button.close').click()
+
+            if (result.rc !== 0) {
+                alert(result.response)
+                return
+            }
+
+            const testExecutionRow = $(`.test-execution-${execution.id}`)
+            animate(testExecutionRow, () => {
+                testExecutionRow.find('.test-execution-hyperlinks').append(renderLink({ is_defect: true, url: result.response }))
```

Comment:
This will add the bug URL even if the bug tracker fails to add it automatically and falls back to manual creation.  To prevent it don't try updating the list of URL here but instead reload the entire TE row unconditionally.